x-pack/plugins/case/server/routes/api/__tests__/delete_case.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  mockCases,
+  mockCasesErrorTriggerData,
+  createMockSavedObjectsRepository,
+  createRouteContext,
+} from '../__fixtures__';
+import { initDeleteCaseApi } from '../delete_case';
+import { IRouter, kibanaResponseFactory } from 'src/core/server';
+import { loggingServiceMock, httpServiceMock, httpServerMock } from 'src/core/server/mocks';
+import { CaseService } from '../../../services';
+import { securityMock } from '../../../../../security/server/mocks';
+
+describe('DELETE case', () => {
+  const setup = async () => {
```

Comment:
I think that `setup` can be a util function that test can use if I do not miss anything here